launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light_node_container.launch.py

```diff
@@ -63,11 +69,21 @@ def add_launch_arg(name: str, default_value=None, description=None):
     # traffic_light_classifier
     add_launch_arg("classifier_type", "1")
     add_launch_arg(
-        "classifier_model_path",
+        "classifier_model_path_car",
```

Comment:
```suggestion
        "car_classifier_model_path",
```
To use the same way as `fine_detector_model_path` or `fine_detector_label_path`
And please check other related parameters.

Reply:
The parameters are fixed in https://github.com/autowarefoundation/autoware.universe/pull/5871/commits/e464e171f253dc9e987bfee7fa60091138f2ab87.